suites/selftest/conf/trc/tools/default_args.trc.xml

```diff
@@ -0,0 +1,8 @@
+<test name="default_args" type="script">
```

Comment:
I see `default_args` name here.  And it is applied to the `defailt_args.c` test.  Is it necessary that these names match?